isis/src/apollo/apps/apollopaninit/apollopantranstable.trn

```diff
@@ -0,0 +1,53 @@
+  Group = SpacecraftName
```

Comment:
I think we should come up with a standard for naming conventions. For the ones I moved, I capitalized the name of the mission and the instrument name.